### DISCUSSION TITLE
install ITK while installing ANTS.

### DISCUSSION THREAD

**Discussion by MRIDude** [ANTsX/ANTs#1417] [General]
I am not an expert in cmake. I would like to get the version of ITK that is compiled during the compiling of ants to be put in the same prefix directory. I am trying to avoid compiling ITK separately.   Is this possible?

**Comment by cookpa** [comment]
That's a good question. ITK is configured to install to `${buildDir}/staging`. Wherever you run cmake from is `$buildDir`. You should see in there `staging` and `ITKv5-build`. 

To reconfigure it, from `ITKv5-build`:

```
cmake -DCMAKE_INSTALL_PREFIX=/path/to/new/installDir .
make install
```

For me this triggers some recompiling (or maybe just relinking). I think this is faster than a full recompile though.

You could also try

`cmake --install . --prefix /path/toinstall`

**Comment by cookpa** [comment]
I tried it out and I think this is the way

```
cd ITKv5-build
cmake --install . --prefix /path/to/install
```
